src/cdk/menu/menu.md

```diff
@@ -10,68 +10,65 @@ directives apply their associated ARIA roles to their host element.
 The directives in `@angular/cdk/menu` set the appropriate roles on their host element.
 
 | Directive           | ARIA Role        |
-| ------------------- | ---------------- |
+|---------------------|------------------|
 | CdkMenuBar          | menubar          |
 | CdkMenu             | menu             |
 | CdkMenuGroup        | group            |
 | CdkMenuItem         | menuitem         |
 | CdkMenuItemRadio    | menuitemradio    |
 | CdkMenuItemCheckbox | menuitemcheckbox |
+| CdkMenuTrigger      | button           |
 
 ### Getting started
 
 Import the `CdkMenuModule` into the `NgModule` in which you want to create menus. You can then apply
 menu directives to build your custom menu. A typical menu consists of the following directives:
 
-- `cdkMenuTriggerFor` - links a trigger button to a menu you intend to open
-- `cdkMenuPanel` - wraps the menu and provides a link between the `cdkMenuTriggerFor` and the
-  `cdkMenu`
-- `cdkMenu` - the actual menu you want to open
-- `cdkMenuItem` - added to each button
+- `cdkMenuTriggerFor` - links a trigger element to an `ng-template` containing the menu the trigger opens
+- `cdkMenu` - creates the actual menu content that the trigger will open
+- `cdkMenuItem` - added to each item in the menu
 
 <!-- example({
-  "example": "cdk-menu-standalone-menu",
-  "file": "cdk-menu-standalone-menu-example.html"
+  "example": "cdk-menu-standalone-trigger",
+  "file": "cdk-menu-standalone-trigger-example.html"
   }) -->
 
 Most menu interactions consist of two parts: a trigger and a menu panel.
 
 #### Triggers
 
-You must add the `cdkMenuItem` and `cdkMenuTriggerFor` directives to triggers like so,
+`cdkMenuTriggerFor` can be added to any button to make it a trigger for the given menu, or any menu
+item to make it a trigger for a submenu. When adding this directive, be sure to pass a reference to
+the template containing the menu it should open. You can toggle the associated menu using a mouse
```

Comment:
```suggestion
the template containing the menu it should open. Users can toggle the associated menu using a mouse
```
I think "you" in this context is the developer writing the code vs. user who's going to use the menu